routes/index.js

```diff
@@ -16,7 +17,11 @@ router.get('/', (req, res) => {
 
 router.use('/auth', auth)
 
-router.use(requireAuth())
+db.Secret.findOne({ env: process.env.NODE_ENV, key: 'JWT_KEY' }, (err, secret) => {
+  if (err) return console.log(err)
+  const key = new Cryptr(process.env.SECRET_KEY).decrypt(secret.value)
+  router.use(jwtAuth.requireAuth(key))
+})
```

Comment:
I'm not sure if this is ok, instead of config object. Which one should be used in this case, @procateodor ?